tests/test_wog.py

```diff
@@ -596,3 +596,13 @@ def test_xaril_toxins():
 	assert raptor.cost == 0
 	kingsblood.play()
 	assert len(game.player1.hand) == 3
+
+
+def test_yshaarj_rage_unbound():
+	game = prepare_game()
+	yshaarj = game.player1.give("OG_042")
+	yshaarj.play()
+	assert len(game.player1.deck) == 26
+	game.end_turn()
+	assert len(game.player1.field) == 2
+	assert len(game.player1.deck) == 25
```

Comment:
This is a flaky test which will fail if there's no minions in the deck. Prepare an empty game and shuffle a specific minion and spell in the deck.

Make sure to also test with an empty deck.